### DISCUSSION TITLE
Update Bloomberg New Grad SWE

### DISCUSSION THREAD

**Discussion by quantumscholar** [ReaVNaiL/New-Grad-2024#466] [General]
Pull request: https://careers.bloomberg.com/job/detail/118309

**Comment by cold4Fun** [comment]
where does it say no sponsorship?

**Comment by nahianreza** [comment]
They sponsor. I interviewed with them last month as an international

**Reply by cold4Fun** [reply]
IC, so I guess they should revise it.
